.gitlab-ci.yml

```diff
@@ -17,18 +17,26 @@ build-server:
   script:
     - yarn install
     - yarn build
+  artifacts:
+    when: always
+    expire_in: 3 days
+    name: ${CI_PROJECT_PATH_SLUG}-${CI_COMMIT_REF_SLUG}-${CI_PIPELINE_ID}-${CI_JOB_NAME}
+    paths:
+      - build/dist
 
 postman-tests:
   stage: test
-  needs: []
+  needs:
+    - build-server
   image: node:16-alpine
   variables:
     ELASTICPATH_CLIENT_ID: $ELASTICPATH_CLIENT_ID
     ELASTICPATH_API_HOST: $ELASTICPATH_API_HOST
-  script:
+  before_script:
     - yarn install
```

Comment:
do we need this yarn install ?

Reply:
In my local test, I couldn't use the build output by itself in a fresh environment. I needed to do `yarn install` for all the runtime dependencies.

There could be more improvements in this area. I haven't looked int the Apollo Server build yet.